docs/content/guides/dapp-deployment/mainnet-deployment.md

```diff
@@ -0,0 +1,70 @@
+
+You can read about Transaction fees here: [https://docs.onflow.org/token/concepts/#fees](https://docs.onflow.org/token/concepts/#fees)
+
+### Ongoing Development
+
+The goal of the Flow project is to become decentralized and community operated, over time. In the early stages of the network the Flow core team will be primarily responsible for the stability of the network, adding protocol features included in the node software...etc. The stability of the network will increase over time, as more nodes join and begin to participate in the protocol, however there are still "unknown-unknowns" (an apt idiom from a regrettable source) Early adopters of Flow are test pilots attempting to break blockchain barriers of the past, and we're asking all early-adopters for their help flagging issues with the Flow core team.
+
+## Deploying on Mainnet
+
+Currently, deploying to Mainnet requires manual intervention from the Flow team. To make your deployment as fast and as smooth as possible please use the steps in the guide to prepare your project.
+
+### Prerequisites
+
+Before considering your application for deployment to Mainnet, it has to have been live on Testnet handling real world traffic for a fixed period of time as suggested here: **Testnet Testing Guidelines.**
+
+Your project's smart contracts should include test coverage and follow any additional guidelines set out here: [Smart Contract Testing Guidelines](../contract-testing).
+
+Your reviewers will be using the following criterion when assessing your project: [Flow Project Technical Analysis](../contract-testing) Keeping these points in mind while developing your project will help speed up the review and deployment process**.**
+
+### Smart Contract Review
+
+To ensure that Flow Mainnet works well for everyone in the early days, all smart contracts being deployed will be reviewed by the Flow team. The aim of the review process is to attempt to discover any issues that may affect your users or the operation of the network before your application is live.
+
+To submit your contract for review use this form: [https://buildwithflow.typeform.com/to/EkfaboAx](https://buildwithflow.typeform.com/to/EkfaboAx)
+
+Once your contract has been submitted for review a member of the reviewing team will contact you and discuss how long the review might take. The timeliness of the review process will be different for every project.
+
+### Deployment Process
+
+When your application is approved for deployment, you'll need to make the Flow team aware of some important information:
+
+- Provide a git tag pointing to the source code of the smart contracts you would like to deploy, so the team can perform a checkout of the correct version.
+- Provide all the required public keys for all necessary Mainnet accounts.
+- For projects requiring multiple contracts, you'll need to provide a deployment script and instructions explaining how to perform your multi-contract deployment correctly.
```

Comment:
```suggestion
- For projects requiring multiple contracts, you'll need to provide a deployment script and/or instructions explaining how to perform your multi-contract deployment correctly.
```
I actually thought this would be and/or